### DISCUSSION TITLE
The apm service uses apollo as the configuration center. The configuration does not take effect

### DISCUSSION THREAD

**Discussion by shining-stars-l** [apache/skywalking#11646] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

When the apm service uses apollo as the configuration center, configuration path filtering does not take effect

### case 1 
- key  configuration-discovery.default.agentConfigurations
- value agent.trace.ignore_path:post:/user/add
### case 2
- key configuration-discovery.default.agentConfigurations
- value agent.trace.ignore_path=post:/user/add
### case 3
- key  configuration-discovery.default.agentConfigurations
- value trace.ignore_path:post:/user/add
### case 4
- key configuration-discovery.default.agentConfigurations
- value trace.ignore_path=post:/user/add
## situation description

The key of the configuration value in apollo configuration value and the configuration of value use the: symbol to connect, but the record path of oap is post:/user/add, so: is not a conflict? So I tried everything to implement the exclusion path, and it still didn't work. Wondering how to configure to exclude post:/user/add in apollo?

### What you expected to happen

I want to configure the path to exclude record post:/user/add in apollo

### How to reproduce

The agent monitors a service and executes a request to the service. The request uses post mode, and then the oap configuration center uses apollo to configure the exclusion record of the request

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Why colon is a conflict? What do you mean? A limitation at the apollo side?

**Reply by shining-stars-l** [reply]
This screenshot is an example of the official skywalking website using apollo. Use a colon between default and 200.
Another screenshot is to filter out the path in the Apollo, oap display path for POST: / userloginstatistics/testdistributedtransaction, this: not conflict

![11111](https://github.com/apache/skywalking/assets/40255310/bab25b3e-2e2b-401a-9280-f765db4807f9)
![2222222](https://github.com/apache/skywalking/assets/40255310/cabc52bf-3339-4711-9e5a-76f982ca7a6f)

**Reply by wu-sheng** [reply]
Our document is not about apollo, we treat all kinds of config servers consistently. 
So, if something goes wrong on a specific server, it is possible. But, I am not totally following your words.

You marked as answer selected, I dont' know whether you mean this has been resolved somehow.

**Reply by shining-stars-l** [reply]
Set as the answer has been selected is that I have not understood the way to ask this question, you mean that this configuration center is unified processing, then I can only go to see the source code to analyze, is this meaning

**Reply by wu-sheng** [reply]
The docs only provide you the key and value. If the configuration server has more requirements about how to set these, such as the rules for specific chars. I have no idea about that.
